spec/schema_tools/hash_spec.rb

```diff
@@ -16,7 +16,15 @@ class Contact
 end
 
 class OneOfDefinition
-  attr_accessor :person
+  attr_accessor :person, :one_of_schema, :all_of_schema, :any_of_schema
+end
+
+class AnyOfDefinition
+  attr_accessor :person, :one_of_schema, :all_of_schema, :any_of_schema
```

Comment:
This is not what I meant. What I'm looking for is the ability to show this in the test:

`nothing` or a `Person` or a `Contact` to an __AnyOf Node__
`Person` or a `Contact` to an __OneOf Node__, but not nothing, and not all of them
`Person` and a `Contact` to an __AllOf Node__, but not nothing, and not one of them

Granted I don't have full context on how this all works, I'm looking for a way to distinguish between these in test cases.